moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -1216,9 +1216,9 @@
            </widget>
           </item>
           <item row="1" column="0">
-           <widget class="QPushButton" name="import_file_button">
+           <widget class="QPushButton" name="import_3d_obj_file_button">
             <property name="text">
-             <string>Import &amp;File</string>
+             <string>Import &amp;Object</string>
```

Comment:
An "object" is imported in both cases. Hence, "File" to indicate that we import from file was correct, wasn't it? Please revert and better use tooltips if you want to be more specific.
```suggestion
             <string>Import &amp;File</string>
```

Reply:
I find the hint that this imports a mesh rather than some other format (a scene? a YAML? some other list of objects?) is hard to discover. I think it should be more visible than a tooltip. How about `Import Mesh`?

To be clear, the problem is not that the button is incorrect, it's that it doesn't convey what a common user would be looking for or understand at first sight. Reading "Import File" as a new user makes me think "What kind of file? Ugh, I'll look into it later", "Import Mesh" makes me think "Oh cool, now I know I can do that".

Reply:
The key point is that both buttons essentially import an object mesh, once from a local file, once from an url.
Hence the button names should distinguish file and url.

Reply:
True, it is a bit of a break in consistency, but I value the discoverability higher. The current state leaves the arguably most important piece of information ("this imports an object mesh") unspoken. 

I would prefer to express the main function on the more common button and let the user discover the secondary option via tooltips/dialogue details. E.g. one button "Import Mesh" with tooltip "Import mesh from file" and one button "From URL" with tooltip "Import mesh from URL".

I am making the silent assumption here that importing via URL is uncommon, but I think it's a reasonable one.

Reply:
The whole GUI block is dealing with objects only as indicated by the frame header "Curren Scene Objects". As said above, changing the button text from file to object obscures the fact, that we will load from a file in contrast to from url.

Reply:
From my experience that's not enough to indicate what the button does. It could also mean "Import Scene Objects", for example. The word "Mesh" is more important than "File" in my opinion, and would make the button a lot more expressive.

Another argument: This `Import File` button in the `Current Scene Objects` frame does not actually import a "Scene Object", it imports a *mesh*, and the button that actually imports a "Scene Object" (or multiple ones) is `Import From Text` in the `Scene Geometry` tab.

I just committed an alternative, will post the pictures below.

Reply:
Suggested change:
![vnew](https://user-images.githubusercontent.com/4535737/85847660-c7c09380-b7e2-11ea-9d1c-98f545a3493e.png)

Original below:
![vold](https://user-images.githubusercontent.com/4535737/85847664-c98a5700-b7e2-11ea-8a41-9d4b8dc8065f.png)

I might want to take out the "Scene" from "Scene Objects" and add a tooltip to "Publish Scene".